~bento-components-1-build.log

```diff
@@ -0,0 +1,745 @@
+alp.max.js
```

Comment:
Does this file need to be merged?

Reply:
No, idk how that snuck in actually. I'll remove that file now